com.unity.netcode.gameobjects/Runtime/Configuration/NetworkConfig.cs

```diff
@@ -266,25 +222,17 @@ public ulong GetConfig(bool cache = true)
                 return m_ConfigHash.Value;
             }
 
-            Sort();
```

Comment:
I think this (+ removal of `.Sort()` method) might potentially break things. (+ @TwoTenPvP for more thoughts)

Reply:
There are no more registered scenes... so there is no need to sort anything.  :)